timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/graphql/datafetchers/SummaryPropDataRetriever.java

```diff
@@ -57,7 +57,7 @@ public SummaryPropDataRetriever(String userConfiguredPredicate, List<SummaryProp
   }
 
   private TypedValue createTypedValue(CursorQuad cursorQuad, DataSet dataSet) {
-    String type = cursorQuad.getValuetype().isPresent() ? cursorQuad.getValuetype().get() : RdfConstants.STRING;
+    String type = cursorQuad.getValuetype().isPresent() ? cursorQuad.getValuetype().get() : RdfConstants.URI;
```

Comment:
Uri is not the default value type, but when the predicate points to another subject (i.e. the triple's object is _not_ a value) this code returns it as if the object _was_ a value of type `xsd:anyURI`

Might be good to put that in the comment as it is not obvious apparently :)